app/Livewire/Project/Database/BackupEdit.php

```
@@ -98,58 +138,169 @@ public function syncData(bool $toModel = false)
+            $localRepo->retention_full_type = $this->localRepoRetentionFullType ?? 'count';
+            $localRepo->retention_full = $this->localRepoRetentionFull ?? 2;
+            $localRepo->retention_diff = $this->localRepoRetentionDiff ?? 7;
+            $localRepo->save();
+            $repoNumber++;
+        } else {
+            $this->backup->pgbackrestRepos()->where('type', 'posix')->delete();
+        }
+
+        if ($hasS3) {
+            if (empty($this->s3RepoStorageId)) {
+                throw new \Exception('S3 Storage is required when enabling S3 backups for pgBackRest.');
```

PHPMD (2.15.0)

Missing class import via use statement (line '283', column '27'). (undefined)

(MissingImport)